Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+KeyCommands.swift

```diff
@@ -216,6 +272,11 @@ extension BrowserViewController {
       UIKeyCommand(input: "]", modifierFlags: .command, action: #selector(goForwardKeyCommand)),
     ]
     
+    navigationCommands += [
+      UIKeyCommand(title: Strings.Hotkey.zoomInTitle, action: #selector(zoomInPageKeyCommand), input: "+", modifierFlags: .command),
+      UIKeyCommand(title: Strings.Hotkey.zoomOutTitle, action: #selector(zoomOutPageKeyCommand), input: "-", modifierFlags: .command)
```

Comment:
are those shortcuts the same as on Safari?

Reply:
Yes it should be equal in Safari